docs/src/development/local/_index.md

```diff
@@ -44,27 +44,30 @@ You will also notice a new directory in your project, `.platform/local`, which i
 
```

Comment:
> ~/htdocs $ platform get [project-id] my-project
Should be:
> $ platform get [project-id] my-project

Reply:
And to be consistent, shouldn't `[project-id]` be `<PROJECT_ID>` , as [here](https://docs.platform.sh/development/troubleshoot.html#clear-the-build-cache).